serverless.yml

```diff
@@ -62,7 +63,7 @@ functions:
           cors: true
           private: ${self:custom.cfg.private}
   onDemandHttpObservatoryScan:
-    handler: handler.addHttpObservatoryScanToQueue
+    handler: lib/httpobsscan_handler.queue
```

Comment:
@claudijd are you OK with this? Basically each function having it's own handler file.
If we do this we probably won't need a generic handler file.

Alternatively we can keep the generic handler file, keep it to a minimum by importing the handler files.

Reply:
I like keeping them separate, I don't need a need to combine if it works.